README.md

```diff
@@ -1,47 +1,185 @@
 # react-e [![CircleCI](https://circleci.com/gh/Originate/react-e.svg?style=svg)](https://circleci.com/gh/Originate/react-e)
 
-React helper to generate elements
+JSX replacement that doesn't need to be compiled.
```

Comment:
Let's brainstorm the tagline a bit more, since it is the thing that will sell this the best. 

What is the primary reason why you built react-e? I doubt that it was because you didn't want to compile stuff. We spoke about how the ideal solution is a compiled DSL that is JSX but better. I also doubt that you did it just to replace JSX, just because. 

What about `JSX alternative with better readability and productivity`?